ConcernsCaseWork/ConcernsCaseWork/Models/Teams/ConcernsTeamCaseworkModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Ardalis.GuardClauses;
+
+namespace ConcernsCaseWork.Models.Teams
+{
+	public class ConcernsTeamCaseworkModel
+	{
+		public ConcernsTeamCaseworkModel(string ownerId, string[] teamMembers)
+		{
+			this.OwnerId = Guard.Against.NullOrWhiteSpace(ownerId);
+			this.TeamMembers = Guard.Against.Null(teamMembers);
```

Comment:
I assume it's fine if the collection is empty?

Reply:
It is fine and to be expected. But will unit tests to prove this is the case going forwards